src/scripts/core/models/column.ts

```diff
@@ -1,23 +1,31 @@
 import { Container } from "pixi.js";
-import { Token } from "./token";
+import { IToken, Token } from "./token";
 import { gsap } from "gsap";
-//
+
 export class Column extends Container {
 
     public tokens: Token[] = [];
-    private columnID: number;
-    private columnSize: number;
-    private availHeight: number;
+    private _columnID: number;
+    private _columnSize: number;
+    private _availHeight: number;
 
-    constructor(columnID: number, columnSize: number, availWidth: number, availHeight: number) {
+    constructor(columnID: number, columnSize: number, availHeight: number) {
         super();
 
-        this.columnID = columnID;
-        this.columnSize = columnSize;
-        this.availHeight = availHeight;
-        for (var tokenIndex: number = 0; tokenIndex < this.columnSize; tokenIndex++) {
-            const newToken = new Token(this.availHeight, this.availHeight, this.columnID, tokenIndex, this.columnSize);
-            newToken.y = (availHeight / (this.columnSize / tokenIndex));
+        this._columnID = columnID;
+        this._columnSize = columnSize;
+        this._availHeight = availHeight;
+
+        const newIToken: IToken = {
```

Comment:
Seems a bit strange to be adding the 'I' to this variable, that's usually reserved for just interface names, to make it clear that it is an interface, heck the const seems unnecessary if you ask me, you can just pass the object directly into 'new token' and it makes things a bit cleaner

Reply:
Kieran seemed unfamiliar with passing an object in directly using an interface as the accepted type so I suggested this may to more easily break it down as what was going on. I agree the naming could change or it simply be passed in directly with an object but for this project this implementation was fine in my opinion.

Reply:
I can understand the logic, I do believe it is worth pointing out the strange naming convention here. Additionally there is a direct modification of this object within the for loop below, which can just be done directly by passing the object in in this way, rather than adding unnecessary lines

Reply:
Again, as this was for learning purposes, the modification was intentional to show how an object can be created and then certain values reassigned without modifying the whole object. Also, the number of lines is irrelevant, we don't want to indicate that more line = bad practice; Have as many lines as needed to make the code readable and understand by the dev.